numpy/distutils/system_info.py

```diff
@@ -92,20 +92,20 @@
 search_static_first = 0
 
 [fftw]
-fftw_libs = rfftw, fftw
-fftw_opt_libs = rfftw_threaded, fftw_threaded
```

Comment:
Why did you remove this line as part of this PR?

Reply:
`fftw_opt_libs` is never referenced anywhere in `system_info.py`, so I removed it due to its connection with `libraries` option

Reply:
interesting. Even when I go back to 575d373479 which added these lines, I do not see how it was ever parsed.

Reply:
Agreed, I couldn't find any comments on it...